modules/fledgeForGpt.js

```diff
@@ -48,5 +47,50 @@ export function addComponentAuctionHook(next, bidRequest, componentAuctionConfig
     logWarn(MODULE, `unable to register component auction config for: ${adUnitCode} x ${seller}.`);
```

Comment:
Should `addComponentAuction` be skipped altogether if FLEDGE is not supported ?
On such a browser, receiving unsolicited auction configs from an adapter will print this misleading under the circumstances warning.

Reply:
The message "unable to do something" tells me the expectation was that it should have been able to, so something went wrong.
If it said "ignored unsolicited fledge configs", that would be more accurate, but I understand we may not want to have a log message for every specific condition